datumaro/cli/commands/explore.py

```diff
@@ -101,24 +110,21 @@ def explore_command(args):
 
     results = explorer.explore_topk(query_datasetitem, args.topk)
 
-    subset_list = []
-    id_list = []
     result_path_list = []
-    log.info("Most similar {} results of query in dataset".format(args.topk))
+    log.info(f"Most similar {args.topk} results of query in dataset")
     for result in results:
-        subset_list.append(result.subset)
-        id_list.append(result.id)
         path = getattr(result.media, "path", None)
         result_path_list.append(path)
-        log.info("id: {} | subset: {} | path : {}".format(result.id, result.subset, path))
-
-    visualizer = Visualizer(dataset, figsize=(20, 20), alpha=0)
-    fig = visualizer.vis_gallery(id_list, subset_list)
+        log.info(f"id: {result.id} | subset: {result.subset} | path : {path}")
 
     if args.save:
-        fig.canvas.draw()
-        data = np.frombuffer(fig.canvas.tostring_rgb(), dtype=np.uint8)
-        data = data.reshape(fig.canvas.get_width_height()[::-1] + (3,))
-        save_image(osp.join("./explorer.png"), data, create_dir=True)
+        saved_result_path = osp.join(args.project_dir, "explore_result")
+        if osp.exists(saved_result_path):
+            os.rmdir(saved_result_path)
```

Comment:
How about to use `shutil.rmtree` instead of `os.rmdir`? `os.rmdir` does not remove directory trees, it can only remove empty directory without any additional files. https://docs.python.org/3/library/os.html#os.rmdir